lib/src/widget/built_in/simple/simple.dart

```diff
@@ -44,28 +46,42 @@ class SimpleToastWidget extends StatelessWidget {
 
   final bool applyBlurEffect;
 
+  final bool? showCloseButton;
+  final VoidCallback? onCloseTap;
+
   SimpleStyle get defaultStyle => SimpleStyle(type);
 
   @override
-  Widget build(BuildContext context) {
-    final background = backgroundColor ?? defaultStyle.backgroundColor(context);
+  _SimpleToastWidgetState createState() => _SimpleToastWidgetState();
```

Comment:
We don't need to make this widget as Statefull we don't have any state in this widget